packages/experiments/src/components/Pagination/examples/Pagination.Buttons.Customization.Example.tsx

```diff
@@ -35,12 +36,10 @@ export class PaginationCustomizationExample extends React.Component<{}, IPaginat
             selectors: {
               ':hover': { backgroundColor: '#c8c8c8', borderRadius: '16px' },
               '&:aria-selected=true': {
```

Comment:
This selector is also _invalid_ (pretty sure) and redefined below. Should be safe to remove...

Reply:
removed

---
In reply to: [241932229](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#discussion_r241932229) [](ancestors = 241932229)